MSAL/src/http/MSALURLSessionDelegate.m

```diff
@@ -57,7 +57,9 @@ - (void)URLSession:(NSURLSession *)session
     
     NSString *requestHost = [request.URL msalHostWithPort];
     
-    LOG_INFO(self.context, @"Redirecting to %@", [MSALAuthority isKnownHost:request.URL] ? requestHost : [requestHost msalShortSHA256Hex] );
+    LOG_INFO(self.context, @"Redirecting to %@", [MSALAuthority isKnownHost:request.URL] ?
+             [NSString stringWithFormat:@"%@://%@", request.URL.scheme, request.URL.host]
```

Comment:
nit: do we actually need scheme here?

Reply:
We might want to look for a scheme in log analyzing, and it doesn't hurt to log here although all traffic should be https.